packages/react/src/components/Cta/index.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+export { default as CTA } from './Cta';
```

Comment:
Again, we can probably keep this all capital, and it'll be ok.

`Cta --> CTA`

Reply:
yeah @photodow when I make change the file name `Cta to CTA ` I am getting an error Module not found: Error: [CaseSensitivePathsPlugin] `/Users/mkothur@us.ibm.com/ibm-dotcom-library/packages/react/src/components/Cta/Cta.js` does not match the corresponding path on disk `CTA.js`.
 @ ./src/components/Cta/__stories__/Cta.stories.js 14:34-51,

Reply:
🤔got it. So it's an eslint error?

Reply:
no its git issue, trying to fix this.

Reply:
this is resolved, it was an import reference issue when I walked through this with @mkothur